### PR TITLE
Performance improvement on application form

### PR DESCRIPTION
**Pull request by theskumar**
This PR optimizes the code and logic responsible for unlocking
the application form on detecting human activity.

- makes the unlocking action DRY.
- move away from using jquery
- prevent the unlock action from firing on every mouse move, the unlock
  action is performed only once.
- Use more standard key names instead of keycodes.

### CODE REVIEW

**Comment by frjo** [comment]
@theskumar  I implemented your performance improvements in to the version I use in my Hugo base theme. That version is vanilla js so I moved does changes over here. Please test and see if I got it right.

**Review by frjo — APPROVED** [review]

File: hypha/static_src/src/javascript/apply/application-form.js [hunk 1]
```diff
@@ -1,72 +1,68 @@
...
+                    .setAttribute("aria-describedby", label_id);
+            }
         }
+    });
 
-        // Remove the "no javascript" messages
-        $(".message-no-js").detach();
+    if (errors.length) {
+        // Set focus to the first error field.
+        errors[0].querySelector("input").focus();
```
Comment (theskumar) [inline hypha/static_src/src/javascript/apply/application-form.js]: Should we use a if condition here. 

```
const inputEl = errors[0].querySelector("input")
if (inputEl) { 
...
}
``

**Comment by theskumar** [comment]
Minor comment to handle edgecase, reset of the jquery -> vanilla JS looks great!

**Comment by theskumar** [comment]
Look good 🚢

**Pull request merged**